docs/3_restarting_and_continuing_a_case.ipynb

```diff
@@ -1,5 +1,18 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "2c3f3225",
+   "metadata": {},
+   "source": [
+    "> [!Warning] \n",
+    "> **This project is still in an early phase of development.**\n",
+    ">\n",
+    "> The [python API](https://c-star.readthedocs.io/en/latest/api.html) is not yet stable, and the schema for the [blueprint](https://c-star.readthedocs.io/en/latest/terminology.html#term-blueprint) will likely evolve. \n",
+    "> Therefore whilst you are welcome to try out using the package, we cannot yet guarantee backwards compatibility. \n",
+    "> We expect to reach a more stable version in Q1 2025."
+   ]
+  },
   {
```

Comment:
Here we should also warn the user that the notebook assumes that they work on perlmutter.

Reply:
yes that's tracked by https://github.com/CWorthy-ocean/C-Star/issues/187